conda/cli/main_info.py

```diff
@@ -345,9 +345,7 @@ def execute(args, parser):
     if (
         args.all or all(not getattr(args, opt) for opt in options)
     ) and not context.json:
-        stdout_logger = getLogger("conda.stdoutlog")
-        stdout_logger.info(get_main_info_str(info_dict))
-        stdout_logger.info("\n")
+        print(get_main_info_str(info_dict) + "\n")
```

Comment:
Interesting way to send messages presumably to stdout with stdout_logger instead of print. I wonder if that flexibility was ever used.

Reply:
I'm curious about this change, the `conda.stdoutlog` logger behaves the same as `print()`?

Reply:
It's effectively the same for the CLI.

I had to change it because the logging was resulting in the output getting collected differently so the new `conda_cli` didn't work correctly for `run` commands.

Per my understanding, an effort was previously undertaken to replace all print (to both stdout and stderr) with this kind of logging logic. My best guess is this was to potentially allow third-party tools (like navigator) to better redirect/interface with the text streams. This effort was never completed and the logging system has become over complicated as a result. (I'll ping the navigator team to check that this won't cause issues)